### PR TITLE
docs: add instructions for executing first script

### PR DESCRIPTION
**Pull request by kevin-brotcke**
Make it a little clearer how to execute your first script when going
through the introduction documentation.

### CODE REVIEW

**Comment by kevin-brotcke** [comment]
I've used other test frameworks where you execute the tests using the binary (in this case that would be `playwright first_script.js`) so it wasn't immediately clear to me you just execute a regular `node` command. Hope this helps others onboard.

File: docs/src/intro-js.md [hunk 1]
```diff
@@ -55,6 +55,12 @@ const { webkit } = require('playwright');
 })();
 ```
 
+Save the script above in a filename of your choosing, such as `first_script.js`. Then execute the script from the command-line using Node.js.
+
+```
+npx node first_script.js
```
Comment (pavelfeldman) [inline docs/src/intro-js.md:61]: Looks like you don't need npx here, `node first_script.js` is sufficient.

**Comment by pavelfeldman** [comment]
Looks like this was dropped.

File: docs/src/intro-js.md (hunk 1 above)
Comment (kevin-brotcke) [reply docs/src/intro-js.md:61]: I wanted to show good practices but it's not necessary for the example. I'll remove it.

**Comment by kevin-brotcke** [comment]
Fixed. Doesn't look like I have access to open this pull request so I created a new one https://github.com/microsoft/playwright/pull/6208

Also turned on GitHub notifications so I don't miss comments. :)

**Pull request closed without merging**